### ISSUE TITLE
doesn't support Markdown nested lists

### ISSUE THREAD

**Issue by wohlben** [josephernest/writing#5]
the nested list syntax doesnt render as expected

1. this should be a list
* this should be another list
  - this should be a sublist
    1. another one with numbers

It just renders as a flat list in Writing. always creating a new list on every unordered / ordered switch.

**Comment by josephernest** [comment]
A solution would be to add a few CSS lines [here](https://github.com/josephernest/writing/blob/master/index.html#L38)  for nested lists with `ul` `li` etc.

If you have an idea, could you post the relevant CSS additions? Thanks!

**Comment by DavideRoss** [comment]
Have you tried to indent with four spaces each line?

```markdown
# Nested lists

* Level 1
    * Level 2
        * Level 3
```

It renders as intended. A fancy addition could be handling the TAB key to add four spaces (currently not working on MacOS 10.12.6).

EDIT: I just discovered that indent is inconsistent: you just need a space before second line to have a second level list, but you have to insert four if you want a third level.

**Comment by wohlben** [comment]
you're right, my initial report was incorrect. sorry for that! (it also works with the expected two spaces)

the nesting only breaks if there are mixed ul/ol lists, and it doesnt detect a nl as a new list. this causes the root ul to fall into the nesting as well


this breaks:
```md
1. test
1. test

* test
    *  test
```
